### ISSUE TITLE
Encrytpt Backups

### ISSUE THREAD

**Issue by rickconlee** [rickconlee/WordPress-Migrator#2]
The backups absolutely 100% need to be encrypted because there is secret information in config files and databases. User needs to be able to choose the key, but we need to make sure that the key we enforce key complexity. 

This task depends on #1

**Comment by rickconlee** [comment]
Marking as wontfix because I need to spend more than a few days of brainpower to figure out how best to handle the encrypted backups where they can also be used locally.
